lib/dish.rb

```diff
@@ -0,0 +1,12 @@
+class Dish
+
+  def initialize(name, quantity, price)
+    @name = name
+    @price = price
+    @quantity = quantity
+  end
+
+  def add
+    { dish: @name, price: @price, quantity: @quantity }
```

Comment:
This feels a little strange to me - you have a class which allows you to create an object with all of the information you need to represent a dish but then have a method to translate that object into a hash containing the same information. I'm not sure what you gain from doing this 🤔

Reply:
That said, I can't actually see anywhere in the code base that this method is being used, so maybe it should just be deleted 🙂

Reply:
I did wonder about this when I was writing it as I thought it was a bit useless, however I just included it because I thought the challenge might want to see how we link the use of the classes together, but I agree that I could just remove it :)